package.json

```diff
@@ -7,15 +7,17 @@
   "main": "./src/index.js",
   "scripts": {
     "test": "mocha",
-    "lint": "eslint **/*.js --ignore-pattern node_modules/"
+    "lint": "eslint **/*.js --ignore-pattern node_modules/",
+    "lint-fix": "eslint **/*.js --ignore-pattern node_modules/ --fix"
```

Comment:
let us not put too many scripts in here, i never trust --fix anyway

Reply:
Alright, I just fixed it.